docs/source/contributing/local-deployment.rst

```diff
@@ -206,16 +196,8 @@ Stopping
 
 To stop the Substra stack, you need to stop the 3 components (backend, orchestrator and frontend) individually.
 
-* Stop the frontend: This action depends on which option you chose during the launch:
-
-  a. Local server: Stop the process running the local server (usually using *Control+C* or *Command+C* on macOS)
-  b. Docker:
-
-     .. code-block:: bash
-
-        docker stop DOCKER_FRONTEND_CONTAINER_NAME
+* Stop the frontend: Stop the process running the local server in Docker (usually using *Control+C* or *Command+C* on macOS)
```

Comment:
On my laptop ⌘+C doesn't work for this purpose and I've always used Ctrl+C -- I thought it was pretty elegant how Ctrl+C always send SIGINT and ⌘+C always copies (compared to the X11 solution).

Reply:
yes, that's indeed the case, I've copied-pasted existing instructions without reading them 🤓